src/components/Firestore/CollectionList.test.tsx

```diff
@@ -32,37 +34,39 @@ jest.mock('./api');
 
 it('shows the list of collections', async () => {
   const fakeApi = new DatabaseApi();
-  fakeApi.getCollections.mockResolvedValueOnce([
-    fakeCollectionReference({ id: 'coll-1' }),
-    fakeCollectionReference({ id: 'coll-2' }),
-  ]);
+  const getCollections = makeDeferred<FakeCollectionReference[]>();
+  fakeApi.getCollections.mockReturnValueOnce(getCollections.promise);
 
-  const { getByText, queryByText } = render(
+  const { getByText } = await render(
```

Comment:
I think this should just be `= render(` ?

Reply:
Oops. Done.